<launch>
	<!-- run the laser_to_pointcloud -->
<!--    <node pkg="easondrone_mapping" type="laser_to_pointcloud.py" name="laser_to_pointcloud"/>-->

    <node pkg="octomap_server" type="octomap_server_node" name="octomap_server">
        <rosparam command="load" file="$(find easondrone_mapping)/config/octomap_server.yaml"/>

        <remap from="/cloud_in" to="/sensor/pc2_out"/>
    </node>
</launch>
